### ISSUE TITLE
Media Permission denied

### ISSUE THREAD

**Issue by modzilla99** [bootc/netbox-chart#52]
When provisioning the volume with NFS for RWX it will give me the permission denied error since by default the folder will be owned by "nobody".

To mitgate this a init container like this could be added:

```yaml
...
  - name: init-chown
    image: busybox:1.32.1
    imagePullPolicy: IfNotPresent
    command:
    - /bin/sh
    - -c
    - chown -R 1000:1000 /mnt/media
    securityContext:
      capabilities:
        drop:
        - ALL
      readOnlyRootFilesystem: true
      runAsNonRoot: false
    volumeMounts:
    - mountPath: /mnt/media
      name: media
...
```

**Comment by bootc** [comment]
You should probably just set the `podSecurityContext.fsGroup` setting to something suitable; Kubernetes should do the rest.

**Comment by rjhenry** [comment]
See #63 - looks to be the same issue.

**Comment by bootc** [comment]
The fix is indeed to set the `fsGroup` in the `podSecurityContext`. Please see https://github.com/bootc/netbox-chart/issues/63#issuecomment-968101116 for how to do this in versions of the chart pre 4.0 (currently unreleased).

**Comment by bootc** [comment]
I've just released version [4.0.1](https://github.com/bootc/netbox-chart/releases/tag/4.0.1) of this chart which resolves this issue.
